src/main/java/io/github/thebusybiscuit/slimefun4/utils/itemstack/ItemStackWrapper.java

```diff
@@ -46,6 +59,17 @@ public ItemStackWrapper(@Nonnull Material material) {
         this(new ItemStack(material));
     }
 
+    public static @Nonnull ItemStackWrapper forceWrapItem(@Nonnull ItemStack itemStack) {
+        return new ItemStackWrapper(itemStack);
+    }
+
+    public static @Nonnull ItemStackWrapper ofItem(@Nonnull ItemStack itemStack) {
+        if (itemStack instanceof ItemStackWrapper) {
+            return (ItemStackWrapper) itemStack;
+        }
+        return new ItemStackWrapper(itemStack);
+    }
```

Comment:
First off, this method is super useful and much needed!
Thanks!

Though I have a few remarks on this in its current state:
1. There are similar static methods for arrays and collections, so I believe these methods should all be grouped together instead of having this one so far up.
2. `ItemStackWrapper.wrap(ItemStack)` seems more adequate to me
3. The nonnull attribute here is not enforced, I also think we should allow null values (just return null)
4. documentation :)

Reply:
Ah, I though the `wrapList` etc was non static for some reason :P. Noted.

Reply:
> The nonnull attribute here is not enforced, I also think we should allow null values (just return null)
Understood. I've been having an issue where there's a lack of a way to tell IDE/static code analysis of the proper nullability contracts. I.e if I were to do this
```java
@Nullable 
public static ItemStackWrapper#wrap(`@Nullable` ItemStack)`
```
My IDE (intellij) tells me that this: 
```java
// the wrapper variable is marked as nullable
ItemStackWrapper wrapper = ItemStackWrapper#wrap(new ItemStack(Material)
```
Jetbrains annotations has an annotation specifically for this called `@Contract`
which would allow me to annotate something like this to denote that the returned
value is null if, and only if, the parameter is null. Is there an equivalent I could use for Slimefun?
```java
@Nullable
@Contract("null -> null") 
public static ItemStackWrapper#wrap(`@Nullable` ItemStack)`
```

Reply:
Should be resolved; however, I'm leaving this comment unresolved so that the annotations stuff can be addressed if necessary.